lib/components/artist/shows/metadata.js

```diff
@@ -31,7 +30,7 @@ class Metadata extends React.Component {
   }
 
   dateAndLocationString(show) {
-    return show.location.city? show.location.city + ', ' + show.exhibition_period : "";
+    return show.location.city? <SerifText style={styles.serifText, {color: 'grey'}}>{show.location.city + ', ' + show.exhibition_period}</SerifText> : null;
```

Comment:
I think that this function will read much better if you make this a if-else block.